src/components/atoms/Avatar/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
```

Comment:
Let's pair pro this component today. There are a few changes I want to do and it'd be easier to pair pro than make comments for each!